cli/local/docker_spec.go

```diff
@@ -183,7 +188,25 @@ func deployPythonContainer(api *spec.API, awsClient *aws.Client) error {
 
 	err = docker.MustDockerClient().ContainerStart(context.Background(), containerInfo.ID, dockertypes.ContainerStartOptions{})
 	if err != nil {
-		return errors.Wrap(err, api.Identify())
+		if api.Compute.GPU > 0 && strings.Contains(err.Error(), "could not select device driver \"\" with capabilities: [[gpu]]") {
+			if _, ok := docker.MustDockerClient().Info.Runtimes["nvidia"]; ok {
+				fmt.Println("retrying API deployment using nvidia runtime because device driver for GPU was not found")
+				hostConfig.Runtime = "nvidia"
+				hostConfig.Resources.DeviceRequests = nil
+				containerCreateRequest, err := docker.MustDockerClient().ContainerCreate(context.Background(), containerConfig, hostConfig, nil, "")
+				if err != nil {
+					return errors.Wrap(err, api.Identify(), "failed to allocate GPU")
```

Comment:
Would it be useful to include the more verbose error message used below for this error and the error on line 202?